README.md

```diff
@@ -21,109 +21,154 @@
+You can also edit a persona on the Streamlit app!
+<img src="static/fortune-teller-edit.png" alt="Screenshot showing the interface for editing a persona on the Streamlit app."/>
+
+### Usage
+
+With `conversant`, you can create a chatbot powered by [Cohere](https://cohere.ai/)'s large language models with just the following code snippet.
+```python
+import cohere
 from conversant import PromptChatbot
 
-client = cohere.Client("<YOUR API KEY HERE>")
-bot = PromptChatbot.from_persona("math-teacher", client=client)
-bot.reply("Hello, World!")
+co = cohere.Client("YOUR_API_KEY_HERE")
+bot = PromptChatbot.from_persona("fantasy-wizard", client=co)
+print(bot.reply("Hello!"))
+>>> "Well met, fair traveller. What bringest thou to mine village?"
 ```
 
-From there, it's also possible to launch the [Streamlit](https://docs.streamlit.io/) app:
-
+You can also define your own persona by passing in your own `ChatPrompt`. 
+```python
+from conversant.prompts import ChatPrompt
+
+shakespeare_config = {
+    "preamble": "Below is a conversation between Shakespeare and a Literature Student.",
+    "example_separator": "<CONVERSATION>\n",
+    "headers": {
+        "user": "Literature Student",
+        "bot": "William Shakespeare",
+    },
+    "examples": [
+        [
+            {
+                "user": "Who are you?",
+                "bot": "Mine own nameth is Shakespeare, and I speaketh in riddles.",
+            },
+        ]
+    ],
+}
+shakespeare_bot = PromptChatbot(
+    client=co, prompt=ChatPrompt.from_dict(shakespeare_config)
+)
+print(shakespeare_bot.reply("Hello!"))
+>>> "Greeteth, and welcome. I am Shakespeare, the great poet, dramatist, and playwright."
 ```
+
+From here, it's also possible to talk to your chatbot using the [Streamlit](https://docs.streamlit.io/) app! This will launch the demo with your chatbot persona pre-selected.
+```python
 from conversant.utils import demo_utils
-demo_utils.launch_streamlit(bot)
+demo_utils.launch_streamlit(shakespeare_bot)
 ```
+## How Conversant Works
+`conversant` uses prompt completion to define a chatbot persona with a description and a few examples. The prompt is sent as input to Cohere's [`co.generate()`](https://docs.cohere.ai/generate-reference/) endpoint for an autoregressive language model to generate text in a few-shot manner from the examples and the current dialogue context. 
 
-## Creating a Custom Persona
-Once you've followed the steps in [Getting Started](#getting-started), you can begin experimenting with creating custom personas! Check out the configuration JSONs for each persona in the [personas directory](/conversant/personas/). You'll need to create a subfolder within this directory that corresponds to your new persona and add a `config.json` file. The directory structure should look like this:
+Each user message and chatbot response is appended to a chat history so that future responses are conditioned on the dialogue context at that point in time.
```

Comment:
Would it be worth pointing readers to the schematics in `CONTRIBUTORS.md` here as well?

Reply:
Added a link to the schematics

[7187638](https://github.com/cohere-ai/sandbox-conversant-lib/pull/41/commits/7187638651b358aab46a953bedb6ee7b87aa807a)